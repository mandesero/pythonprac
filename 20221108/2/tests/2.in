iwjuefb
oawuejrgfn
vdfiok
(0,0), (-1, 0), (0, -100___)
(0,0), (-1, 0), (0, -100)
