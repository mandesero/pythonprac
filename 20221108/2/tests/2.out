InvalidInput
InvalidInput
InvalidInput
InvalidInput
50.00
